src/masternodes/mn_checks.cpp

```diff
@@ -1121,11 +1121,14 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
         }
 
         const auto symbolLength = height >= consensus.FortCanningHeight ? CToken::MAX_TOKEN_POOLPAIR_LENGTH : CToken::MAX_TOKEN_SYMBOL_LENGTH;
+        if (pairSymbol.length() > symbolLength) {
```

Comment:
For safety this should be done by fork height, after the fork if full sync is successful we can remove the fork height check.

Reply:
Probably we don't what it at consensus level, rpc level is better.

Reply:
I'll add checks at rpc level :100: